task/02-numbers-tasks.js

```diff
@@ -38,7 +37,8 @@ function getRectangleArea(width, height) {
  *   0    => 0
  */
 function getCicleCircumference(radius) {
-    throw new Error('Not implemented');
+    let p = 3.141592653589793238462643;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI

Reply:
i got it :) 
thx a lot